test/unit/spread.spec.ts

```diff
@@ -5,6 +5,14 @@ import * as util from "../util";
 const expectUnpack: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toMatch(/[^.]unpack\(/);
 const expectTableUnpack: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toContain("table.unpack");
 
+const spreadCases = [
```

Comment:
It's actually also the case with `tupleReturn` call spread, not sure how it is handled by the current implementation: https://typescripttolua.github.io/play#code/A4JwlgdgLgFAUAAiQgdGqBDARgGwKYoCuEwGAxgNYwDaAjADQIBMjAzALoCU9iyaKmXAWKlKNACyMArIwBsXHpyA